frontend/packages/knative-plugin/src/components/overview/__tests__/EventSinkServicesOverviewList.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import * as utils from '@console/internal/components/utils';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { ResourceLink } from '@console/internal/components/utils';
+import { ServiceModel } from '@console/knative-plugin';
+import EventSinkServicesOverviewList from '../EventSinkServicesOverviewList';
+import { mockEventSourceData } from '../__mocks__/overview-knative-mock';
+
+describe('EventSinkServicesOverviewList', () => {
+  it('should component exists', () => {
+    const wrapper = shallow(<EventSinkServicesOverviewList obj={mockEventSourceData} />);
+    expect(wrapper.exists()).toBeTruthy();
+  });
```

Comment:
How is this a useful test? It's asserting that the wrapper exists. I don't believe there is a way to ever make this fail.